src/patternfly/components/Masthead/masthead.scss

```
@@ -0,0 +1,220 @@
+  --pf-c-masthead--m-display-stack__main--Order: -1;
+  --pf-c-masthead--m-display-stack__main--FlexBasis: 100%;
+  --pf-c-masthead--m-display-stack__main--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__main--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__main--MarginRight: 0;
+  --pf-c-masthead--m-display-stack__main--before--BorderBottom: var(--pf-c-masthead__main--before--BorderBottomWidth) solid var(--pf-c-masthead__main--before--BorderBottomColor);
+  --pf-c-masthead--m-display-stack__content--Order: 1;
+  --pf-c-masthead--m-display-stack__content--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__content--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-masthead--m-display-stack__content--MarginLeft: 0;
+  --pf-c-masthead--m-display-stack__content--c-nav--m-horizontal--MarginRight: calc(var(--pf-c-masthead--inset) * -1);
+  --pf-c-masthead--m-display-stack__content--c-nav--m-horizontal--MarginLeft: calc(var(--pf-c-masthead--inset) * -1);
+
+  // inline
+  --pf-c-masthead--m-display-inline--FlexWrap: nowrap;
+  --pf-c-masthead--m-display-inline__main--Order: 0;
+  --pf-c-masthead--m-display-inline__main--FlexBasis: auto;
+  --pf-c-masthead--m-display-inline__main--PaddingTop: 0;
+  --pf-c-masthead--m-display-inline__main--PaddingBottom: 0;
+  --pf-c-masthead--m-display-inline__main--MarginRight: calc(var(--pf-global--gutter--md) / 2); // split gutter
+  --pf-c-masthead--m-display-inline__main--before--BorderBottom: 0;
+  --pf-c-masthead--m-display-inline__content--Order: 0;
+  --pf-c-masthead--m-display-inline__content--PaddingTop: 0;
+  --pf-c-masthead--m-display-inline__content--PaddingBottom: 0;
+  --pf-c-masthead--m-display-inline__content--MarginLeft: calc(var(--pf-global--gutter--md) / 2); // split gutter
+  --pf-c-masthead--m-display-inline__content--c-nav--m-horizontal--MarginRight: 0;
+  --pf-c-masthead--m-display-inline__content--c-nav--m-horizontal--MarginLeft: 0;
+
+  // main border
+  --pf-c-masthead__main--before--Right: calc(var(--pf-c-masthead--inset) * -1);
+  --pf-c-masthead__main--before--Left: calc(var(--pf-c-masthead--inset) * -1);
+  --pf-c-masthead__main--before--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-masthead__main--before--BorderBottomColor: var(--pf-global--palette--black-600);
+
+  // toggle
+  --pf-c-masthead__toggle--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-masthead__toggle--MarginLeft: calc(var(--pf-global--spacer--md) * -1); // offset by button padding
+
+  // light 100
+  --pf-c-masthead--m-light--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-masthead--m-light__main--BorderBottomColor: var(--pf-global--palette--black-200);
+
+  // light 200
+  --pf-c-masthead--m-light-200--BackgroundColor: var(--pf-global--BackgroundColor--200);
+  --pf-c-masthead--m-light-200__main--BorderBottomColor: var(--pf-global--palette--black-300);
```

Same here - would the border color var work?

```suggestion
  --pf-c-masthead--m-light-200__main--BorderBottomColor: var(--pf-global--BorderColor--100);
```